requirements-pinned.txt

```diff
@@ -31,7 +31,7 @@ cryptography==2.9         # via securesystemslib
 enum34==1.1.10             # via cryptography
 ipaddress==1.0.23         # via cryptography
 iso8601==0.1.12
-pathspec==0.7.0
+pathspec==0.8.0
```

Comment:
Would you kindly rebase your branch on the most recent `develop`? This should make this line go away.